modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImpl.java

```diff
@@ -384,6 +415,94 @@ public float getOwnLoad() {
     return localSystemLoad;
   }
 
+  @Override
+  public double getHardwareLoad() {
+    OperatingSystemMXBean osBeanJava = ManagementFactory.getPlatformMXBean(OperatingSystemMXBean.class);
```

Comment:
So how does this handle cases where my current container context doesn't have access to all the host CPUs, but its load values still show the full system load?  Ie, my host right now has a load of 2.4, but my LXC container only gets two cores, and isn't actually doing anything right now anyway.  In theory it should accept jobs, but the code here wouldn't allow that if it's detecting two cores, and would allow far too much load if it's detecting all 16 cores.